setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+extend-ignore = E203
```

Comment:
Can the configuration here be included in the `pyproject.toml` instead, or does flake8 not support that yet?

Reply:
flake8 does not support it yet, afaik, if you have any resource that says otherwise, please I'd be more than happy to move this

Reply:
You're [right](https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations) 😢